orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/WaitForDisabledServerGroupTask.java

```diff
@@ -64,6 +69,23 @@ public TaskResult execute(@NotNull StageExecution stage) {
     }
   }
 
+  private boolean isPartialDisable(Object desiredPercentage) {
```

Comment:
Could you add a comment here such that we know this behavior is for rolling redblack and monitored deployment strategies?

Reply:
ya good point

Reply:
in 13c608c5310f3dfc4c96cf11ad53907c4fbc4378